src/graph/optimizer/OptimizerUtils.cpp

```diff
@@ -850,10 +850,30 @@ void OptimizerUtils::eraseInvalidIndexItems(
   }
 }
 
+bool OptimizerUtils::isBestIndex(const nebula::meta::cpp2::IndexItem& indexItem,
```

Comment:
Is it intended to reduce the overhead of lookup the data?

Reply:
yes, it can avoid `IndexEdgeNode` or `IndexTagNode` of execution plan nodes in the storaged, thus reducing the overhead of lookup the data